docs/running_in_particular_vpc_subnet.md

```diff
@@ -0,0 +1,47 @@
+# Running jobs in a particular VPC & Subnetwork
+
+To run in a particular VPC and subnetwork you have to make 3 changes. Convert the vcf_to_bigquery.yaml file to the v2alpha1 api, pass the network and subnetwork to the vcf_to_bq command, and set the network, and subnetwork on the virtual machine.
```

Comment:
Would it be ok to wait until Issue #222 is fixed before merging this so that we have a consistent v2alpha1 story?

Reply:
FYI, I updated our docs to use v2alpha1, but it seems like we still need to use the custom yaml file as gcloud doesn't have network/subnetwork options :/